### PR TITLE
Adding and removing a few professors for Clemson University

### PR DESCRIPTION
**Pull request by chengguo-hci**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

0. Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv"). **IF YOU DO NOT DO THIS, YOUR COMMIT WILL BE SUMMARILY REJECTED.**

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

2. _Insert new faculty **in alphabetical order**, not at the end of `csrankings-[a-z].csv`._ **Do not modify `csrankings.csv`, which is auto-generated.** **DO NOT MODIFY `csrankings-[0-9].csv`, WHICH ARE BEING DEPRECATED. USE `csrankings-[a-z].csv` INSTEAD.**

3. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

**(Advanced) Quick contribution via a shallow clone** 

A full clone of the CSrankings repository is almost 2GB.  To contribute a change without creating a full local clone of the CSrankings repo, you can do a _shallow clone_.  To do so, follow these steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)
2. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.
3. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to do another contribution and some time has passed, perform steps 1-3 again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-d.csv [hunk 1]
```diff
@@ -59,6 +59,7 @@ Dale F. Reed,University of Illinois at Chicago,https://sites.google.com/site/dal
 Dale R. Thompson,University of Arkansas,https://www.netgeekdr.com,NOSCHOLARPAGE
 Dale Reed,University of Illinois at Chicago,https://sites.google.com/site/dalereed,NOSCHOLARPAGE
 Dale Schuurmans,University of Alberta,https://webdocs.cs.ualberta.ca/~dale,xaQuPloAAAAJ
+DAljit Singh Dhillon,Clemson University,https://djsingh.people.clemson.edu/,3GWWmEsAAAAJ
```
Comment (emeryberger) [inline csrankings-d.csv]: Mis-capitalized and needs another entry to account for DBLP alias. https://dblp.org/pid/14/7591.html

File: csrankings-n.csv [hunk 2]
```diff
@@ -291,6 +291,7 @@ Ni Trieu,Arizona State University,https://nitrieu.github.io,esITaOoAAAAJ
 Nian-Feng Tzeng,University of Louisiana - Lafayette,https://people.cmix.louisiana.edu/tzeng,6zq-c5cAAAAJ
 Niangjun Chen,SUTD,https://istd.sutd.edu.sg/people/faculty/niangjun-chen,mOmPukcAAAAJ
 Nianwen Xue,Brandeis University,http://www.cs.brandeis.edu/~xuen,KlVbOkMAAAAJ
+Nianyi Li,Clemson University,https://www.clemson.edu/cecas/departments/computing/people/faculty/computer_science.html,3SqliH4AAAAJ
```
Comment (emeryberger) [inline csrankings-n.csv]: Needs to be the faculty member's home page.

File: csrankings-y.csv [hunk 3]
```diff
@@ -291,6 +291,7 @@ Yiming Hu,University of Cincinnati,http://www.ece.uc.edu/~yhu,NOSCHOLARPAGE
 Yiming Yang,Carnegie Mellon University,http://www.cs.cmu.edu/~./yiming,MlZq4XwAAAAJ
 Yin Pan,Rochester Institute of Technology,https://www.rit.edu/gccis/computingsecurity/people/yin-pan,SHJ98SkAAAAJ
 Yin Tat Lee,University of Washington,http://yintat.com,nwHfwCIAAAAJ
+Yin Yang 0002,Clemson University,https://yangzzzy.github.io,-z2_nggAAAAJ
```
Comment (emeryberger) [inline csrankings-y.csv:295]: Not listed here: https://www.clemson.edu/cecas/departments/computing/people/faculty/computer_science.html

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-y.csv (hunk 3 above)
Comment (chengguo-hci) [reply csrankings-y.csv:295]: Listed here: https://www.clemson.edu/cecas/departments/computing/people/faculty/index.html

File: csrankings-d.csv (hunk 1 above)
Comment (chengguo-hci) [reply csrankings-d.csv]: Updated.

File: csrankings-n.csv (hunk 2 above)
Comment (chengguo-hci) [reply csrankings-n.csv]: Updated.

File: csrankings-n.csv [hunk 4]
```diff
@@ -408,6 +407,7 @@ Nicu Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niculae Sebe,University of Trento,http://disi.unitn.it/~sebe,tNtjSewAAAAJ
 Niels A. Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
 Niels Anne Taatgen,University of Groningen,http://www.ai.rug.nl/~niels,F5FI6uYAAAAJ
+Niels Henze,University of Stuttgart,http://nhenze.net,dbVIOtMAAAAJ
```
Comment (emeryberger) [inline csrankings-n.csv]: This entry needs to be removed.

File: csrankings-n.csv [hunk 5]
```diff
@@ -80,7 +80,6 @@ Nancy S. Pollard,Carnegie Mellon University,http://www.cs.cmu.edu/~nsp,Lsr_-hsAA
 Nandita Bhattacharjee,Monash University,http://monash.edu/research/explore/en/persons/nandita-bhattacharjee(a68bc99c-8c24-4087-8a99-b796b7236f70).html,NOSCHOLARPAGE
 Nandita Vijaykumar,University of Toronto,http://www.cs.toronto.edu/~nandita,y7erynoAAAAJ
 Nandlal L. Sarda,IIT Bombay,https://www.cse.iitb.ac.in/~nls,NOSCHOLARPAGE
-Nanning Zheng 0001,Xi'an Jiaotong University,http://en.xjtu.edu.cn/info/1018/1640.htm,iqMe3p8AAAAJ
```
Comment (emeryberger) [inline csrankings-n.csv]: Please remove non-Clemson updates from this PR.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Review by emeryberger — APPROVED** [review]

**Pull request merged**